### PR TITLE
use alan compiler in repo to build anycloud cli

### PR DESCRIPTION
**Pull request by depombo**

### CODE REVIEW

File: anycloud/cli/build.rs [hunk 1]
```diff
@@ -1,29 +1,19 @@
-// This build script depends on `alan` being in the $PATH and compiles the anycloud.ln file so it
+// This build script depends on `alan-compile` being build and compiles the anycloud.ln file so it
```
Comment (dfellis) [inline anycloud/cli/build.rs:1]: Nit `build` `->` `built`

**Comment by dfellis** [comment]
Do we want to edit [.version.sh](https://github.com/alantech/alan/blob/main/.version.sh) in this PR to make sure `anycloud`'s version number is synced with Alan?

**Comment by depombo** [comment]
> Do we want to edit [.version.sh](https://github.com/alantech/alan/blob/main/.version.sh) in this PR to make sure `anycloud`'s version number is synced with Alan?

ah yes!

**Review by aguillenv — APPROVED** [review]

File: .version.sh [hunk 2]
```diff
@@ -12,6 +12,7 @@ echo Version: ${VERSION}
 JSRUNTIME="$(jq ".version = \"${VERSION}\"" js-runtime/package.json)" && echo "${JSRUNTIME}" > js-runtime/package.json
 COMPILER="$(jq ".version = \"${VERSION}\"" compiler/package.json)" && echo "${COMPILER}" > compiler/package.json
 AVM="$(sed "s/^version = .*$/version = \"${VERSION}\"/" avm/Cargo.toml)" && echo "${AVM}" > avm/Cargo.toml
+ANYCLOUD="$(sed "s/^version = .*$/version = \"${VERSION}\"/" anycloud/cli/Cargo.toml)" && echo "${ANYCLOUD}" > anycloud/cli/Cargo.toml
```
Comment (dfellis) [inline .version.sh:15]: These two changes are unfortunately not enough, you also need to run `cargo build` similar to below to make sure the `Cargo.lock` file is updated with the new version.

Comment (depombo) [reply .version.sh:15]: got catch

**Review by dfellis — APPROVED** [review]

**Pull request merged**